Makefile

```diff
@@ -78,4 +78,27 @@ fmt: node_modules
 test: deno
 	go test -v
 
+
+DOCKER_DEV_IMAGE ?= ry/deno:dev-image-0.1
```

Comment:
if you become the owner of this image in Docker Hub then can you change the owner? I wish you were using TravisCI `TRAVIS_REPO_SLUG` instead

Reply:
No, this allows you to use custom build and publish that to any repo (maybe a private registry) and only the user `ry` can push this image to docker hub.

Not sure what's the intention here:
- use a container to run the tests in CI
- publish the docker image
- both

From my use case, I just want a reproducible way to build deno without polluting my machine with all these dependencies.

Reply:
Publishing a "dev" image to Dockerhub and using the published image to run the test would be ideal. Thanks to Docker layering it will speed up test runs a lot. 

https://docs.travis-ci.com/user/build-stages/share-docker-image/

Typically on each release (git tag) on master we want to update the primary docker image in Dockerhub.